src/services/aws/aws.ts

```diff
@@ -127,7 +127,7 @@ export const aws = createApi({
         error ? [] : ["endowments", "profile", "walletProfile"],
       query: (payload) => {
         return {
-          url: `/${v(3)}/profile/${network}/endowment`,
+          url: `/${v(3)}/profile/endowment`,
```

Comment:
```suggestion
          url: `/${v(1)}/profile/endowment`,
```
As per ser Paolo's [comment](https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2483#discussion_r1395103450):
> This endpoint will be deprecated and the new one should be `DELETE | PUT /v1/profile/endowment` or `DELETE | PUT`

Reply:
Oh! Good catch!

Reply:
fixed https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2483#:~:text=edit%20profile%20version-,901e642,-ap%2Djustin%20dismissed